src/main/kotlin/no/nav/brukerdialog/ytelse/pleiepengersyktbarn/søknad/api/domene/k9Format/K9Arbeidstid.kt

```diff
@@ -5,15 +5,16 @@ import no.nav.brukerdialog.ytelse.pleiepengersyktbarn.søknad.api.domene.Pleiepe
 import no.nav.k9.søknad.felles.type.Organisasjonsnummer
 import no.nav.k9.søknad.ytelse.psb.v1.arbeidstid.Arbeidstaker
 import no.nav.k9.søknad.ytelse.psb.v1.arbeidstid.Arbeidstid
-
 import java.time.LocalDate
 
 internal fun PleiepengerSyktBarnSøknad.byggK9Arbeidstid(): Arbeidstid {
     val arbeidstid = Arbeidstid().apply {
 
         if(arbeidsgivere.isNotEmpty()) medArbeidstaker(arbeidsgivere.tilK9Arbeidstaker(fraOgMed, tilOgMed))
 
-        medFrilanserArbeidstid(frilans.k9ArbeidstidInfo(fraOgMed, tilOgMed))
+        if (frilans.arbeidsforhold != null) {
```

Comment:
Jeg tror dette skal virke bra. Fra søknadsdialogen har vi 3 typer frilansdata som sendes inn:
```
type FrilansApiData =
    | FrilansApiDataIngenInntekt
    | FrilansApiDataKunHonorarMisterIkkeHonorar
    | FrilanserMedArbeidsforholdApiDataPart;
```
hvor kun `FrilanserMedArbeidsforholdApiDataPart` sender med `arbeidsforhold`. Arbeidsforholdet er da: 
```
 interface ArbeidsforholdApiData {
    normalarbeidstid: NormalarbeidstidApiData;
    arbeidIPeriode?: ArbeidIPeriodeApiData;
}

```

Reply:
Riktig. Da med `FrilansApiDataIngenInntekt` ikke har arbeidsforhold ønsker man heller ikke å mappe på 0/0 timer. 
For de øvrige blir mapping som vanlig.